GX Map (#3)
genexuslabs/web-controls-library: src/components/map-marker/readme.md

# gx-map-marker

This element represents a marker inside a `<gx-map>` element.

> `coords` attribute must always have a value. If there is not any value set, the marker will be initialize with default values and it will be drawn at `0, 0` coordinates.

Author: I do not understand the suggestion, the comment should be like:

`coords` attribute must always have a value. If value is **not** specified, the marker will be initialized with default values and it will be drawn at `0, 0` coordinates.

> `coords` attribute must always have a value. If there is not any value set, the marker will be initialize with default values and it will be drawn at `0, 0` coordinates.

Author: Ready then!

<!-- Auto Generated Below -->

## Properties

| Property         | Attribute         | Description                                                          | Type     |
| ---------------- | ----------------- | -------------------------------------------------------------------- | -------- |
| `coords`         | `coords`          | The coordinates where the marker will appear in the map.             | `string` |

> | `iconSrc`        | `icon-src`        | The URL of the marker image. _Note: The ideal image size is 25 x 41_ | `string` |

Author: Ready!

# gx-map-marker

> This web component must be included inner _GX Map_ Component (inner `<gx-map>`).

Reviewer: ```suggestion
This element represents a marker inside a `<gx-map>` element.
```

Author: Ready.

> `coords` attribute must always be included in the the tag. If you do not set any attribute, the marker will initialize with default values and it will be draw in `0, 0` position of the map.

Reviewer: ```suggestion
`coords` attribute must always have a value. If a value is not set, the marker will be initialized with default values and it will be drawn at `0, 0` coordinates.
```

Author: Ready!
